### ISSUE TITLE
Wind chart creator overview

### ISSUE THREAD

**Issue by mdschulte** [golf-clash-notebook/golf-clash-notebook.github.io#229]
I have just discovered and used the wind chart creator.  It seems super cool, I had been creating a chart like that by hand every time I upgraded a level.

I do not, however, understand the two columns under each heading (max mid min).  Could you possibly label the chart or at least post an annotated one with explanations?

**Comment by cranst0n** [comment]
@mdschulte yes thats a fine idea. Just to answer your question for the short term, column 1 is rings 1-5, column 2 is rings 6-10.

**Comment by mdschulte** [comment]
When ever do you see 6-10? I've only ever seen 5 rings.

**Comment by cranst0n** [comment]
In higher tours it's pretty common to adjust more than 5 rings because the winds are so high.
